content/en/docs/instrumentation/js/exporters.md

```diff
@@ -45,6 +52,13 @@ const sdk = new opentelemetry.NodeSDK({
     // optional - collection of custom headers to be sent with each request, empty by default
     headers: {},
   }),
+  metricReader: new PeriodicExportingMetricReader({
+    exporter: new OTLPMetricExporter({
+      url: '<your-otlp-endpoint>/v1/metrics', // url is optional and can be omitted - default is http://localhost:4318/v1/metrics
+      headers: {}, // an optional object containing custom headers to be sent with each request
+      concurrencyLimit: 1, // an optional limit on pending requests
```

Comment:
```suggestion
```
We could leave this out as to be consistent with the trace exporter example above :slightly_smiling_face: